mm2src/coins/tendermint/tendermint_token.rs

```diff
@@ -620,7 +622,14 @@ impl MmCoin for TendermintToken {
             let account_info = platform.account_info(&account_id).await?;
 
             let tx = platform
-                .any_to_transaction_data(maybe_pk, msg_payload, &account_info, fee, timeout_height, memo.clone())
+                .any_to_transaction_data(
+                    maybe_priv_key,
+                    msg_payload,
+                    &account_info,
+                    fee,
+                    timeout_height,
+                    memo.clone(),
```

Comment:
not directly related but I think it worth mention that we can avoid cloning memo everywhere.
We can ref str to `tx::Body::new(vec![tx_payload], &memo, timeout_height as u32);` like here
<img width="788" alt="Screenshot 2025-01-24 at 08 07 16" src="https://github.com/user-attachments/assets/1c21abd8-ade9-49c6-9926-cbef0ff8b18d" />